src/lowering/desugar.ml

```diff
@@ -983,18 +983,18 @@ let import_compiled_class (lib : S.comp_unit) wasm : import_declaration =
 let import_prelude prelude : import_declaration =
   decs prelude.it
 
-let inject_decs extra_ds u =
+let inject_decs extra_ds =
```

Comment:
That's fine, but I actually prefer the explict match because then you see the arity of the function in the header.
I wonder if this actually produces more allocation, compiling to a proper curried function rather and a fast 2-argument Ocaml function (not that  it matters here, but it could eslewhere).